sycl/test/xocc_tests/edge_detection/edge_detection.cpp

```diff
@@ -92,10 +101,10 @@ int main(int argc, char* argv[]) {
     // to parallel_fors with local sizes
      [=]() {
       auto gX = xilinx::partition_array<char, 9,
-                xilinx::partition::complete<0>>({-1, 0, 1, -2, 0, 2, -1, 0, 1});
+                xilinx::partition::complete<1>>({-1, 0, 1, -2, 0, 2, -1, 0, 1});
```

Comment:
Curious this change.
Are you changing the API?
Do you think it is better to start at 1?